tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive/inline_images.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Inline png images in html.
+
+Replaces %*.png% in the input .html file with a data URI containing the
+image content of the corresopnding .png files in the images/ subdirectory.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import base64
+import os
+import sys
+
+MIME_TYPE = "image/png"
+
+
+def inline_images():
+    input_html_path = sys.argv[1]
+    image_files = sys.argv[2:]
+    with open(input_html_path, "r") as f:
+        html = f.read()
+    for image_file in image_files:
+        with open(image_file, "rb") as f:
+            base64_content = base64.b64encode(f.read()).decode("utf-8")
+        data_uri = "data:%s;base64,%s" % (MIME_TYPE, base64_content)
+        image_basename = os.path.basename(image_file)
+        image_template = "%%%s%%" % image_basename
```

Comment:
%%wow%% :) Is there other ways to make it more readable? Perhaps `%{}%.format()`?

Reply:
`"%" + image_basename + "%"` does it. Done.